server/led_sign/led_sign_server.py

```diff
@@ -14,6 +15,7 @@ class LedSignServicer(led_sign_pb2_grpc.LedSignServicer):
     CURRENT_DIRECTORY = os.path.dirname(os.path.abspath(__file__)) + sep
     proc = None
     sign_data = {}
+    visual = LedSignMock()
```

Comment:
I might be wrong, but I think that you should initialize your class when the server is being run? so this line would go inside the `serve` method

Reply:
i did it this way because of the comment above, why would it go in serve?

Reply:
Since your class is acting as another LED sign, we want to make sure it's running as long as the server itself is running. This way the information inside of it will always persist as long as the server is up